### PR TITLE
Tidy up validate-modules ignores for modules: cloud/centurylink (rebased)

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Tidy up on sanity tests' ignore files, removing lines with validation-modules for these modules.

The focus was to pass the tests so no code was changed other than adjusting argument_spec for missing types or inconsistencies with the docs. The priority has been not to change any functionality, but to ensure that code and docs are synchronized.

Some lines were added back to the files, but a total of 77 lines were removed (among 2.9, 2.10 and 2.11).

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
plugins/modules/cloud/centurylink/clc_server_snapshot.py
plugins/modules/cloud/centurylink/clc_server.py
plugins/modules/cloud/centurylink/clc_publicip.py
plugins/modules/cloud/centurylink/clc_modify_server.py
plugins/modules/cloud/centurylink/clc_loadbalancer.py
plugins/modules/cloud/centurylink/clc_group.py
plugins/modules/cloud/centurylink/clc_blueprint_package.py
plugins/modules/cloud/centurylink/clc_firewall_policy.py
plugins/modules/cloud/centurylink/clc_alert_policy.py
plugins/modules/cloud/centurylink/clc_aa_policy.py

### CODE REVIEW

**Comment by felixfontein** [comment]
Continuation of #1360.

File: plugins/modules/cloud/centurylink/clc_firewall_policy.py [hunk 1]
```diff
@@ -16,45 +16,54 @@
...
     description:
       - The list of destination addresses for traffic on the terminating firewall.
         This is required when state is 'present'
+    type: list
   ports:
     description:
       - The list of ports associated with the policy.
         TCP and UDP can take in single ports or port ranges.
-    choices: ['any', 'icmp', 'TCP/123', 'UDP/123', 'TCP/123-456', 'UDP/123-456']
+      - "Example: I(['any', 'icmp', 'TCP/123', 'UDP/123', 'TCP/123-456', 'UDP/123-456'])"
```
Comment (felixfontein) [inline plugins/modules/cloud/centurylink/clc_firewall_policy.py:41]:
```suggestion
      - "Example: C(['any', 'icmp', 'TCP/123', 'UDP/123', 'TCP/123-456', 'UDP/123-456'])."
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks for this one as well!

**Comment by russoz** [comment]
@felixfontein thank you for reviewing!

**Pull request merged**